src/go/rpk/pkg/cli/cluster/partitions/unsafe_recover.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2023 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package partitions
+
+import (
+	"fmt"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/adminapi"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func newUnsafeRecoveryCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+	var (
+		dry       bool
+		noConfirm bool
+		nodes     []int
+	)
+	cmd := &cobra.Command{
+		Use:   "unsafe-recover",
+		Short: "Recover unsafely from partitions that have lost majority",
+		Long: `Recover unsafely from partitions that have lost majority.
+
+This command allows you to unsafely recover all data adversely affected by the 
+loss of the nodes specified in the '--from-nodes' flag. This operation is unsafe 
```

Comment:
nit: perhaps would be nice to mention that it may result in a data loss (clarifying what unsafe is).